cost-analyzer/pod-utilization.json

```diff
@@ -781,13 +917,12 @@
     ]
   },
   "time": {
-    "from": "now-24h",
-    "to": "now"
+    "from": "2021-03-22T01:33:44.520Z",
```

Comment:
Doesn't seem right. can we rever this to now-24h?

Reply:
Should now be fixed